docs/DeveloperGuide.md

```diff
@@ -736,17 +735,33 @@ testers are expected to do more *exploratory* testing.
 
     1. Download the jar file and copy into an empty folder.
 
-    1. Double-click the jar file Expected: Shows the GUI with a set of sample food items. The window size may not be
+    2. Double-click the jar file Expected: Shows the GUI with a set of sample food items. The window size may not be
        optimum.
 
```

Comment:
Would it be good to add a note saying if double click dont work can use the java -jar command from terminal?